app/models/miq_group.rb

```diff
@@ -144,6 +144,26 @@ def set_filters(type, filter)
     self.filters[type.to_s] = filter
   end
 
+  def self.remove_all_managed_filters(filter)
```

Comment:
I think the name of this method could be misleading. It sounds like it would remove *all* managed filters. Maybe
```
def remove_tag_from_all_managed_filters(tag)
```